test/fixtures/parser/postgres/postgres_select.sql

```diff
@@ -0,0 +1 @@
+SELECT timestamp with time zone '2005-04-02 12:00:00-07' + interval '1 day';
```

Comment:
Not can we add a newline? (those red marks in Github really bug me! 😁)

```suggestion
SELECT timestamp with time zone '2005-04-02 12:00:00-07' + interval '1 day';

```

Reply:
Done